IMC_WebDev/MiniCityBlog.md

```diff
@@ -0,0 +1,223 @@
+## Building a Mini City: HTML Portion
+## Purpose
+HTML and CSS are extremely important parts of learning about how websites work. Every website uses HTML in order to display things such as text or images on sites. Being able to grasp a few concepts of  HTML and CSS can help a lot when it comes to learning more intricate subjects in computer science. For this reason, it is important to get some experience with it!
```

Comment:
Optional, but I would suggest phrasing the first sentence as "HTML and CSS are extremely important in learning how websites work" and omitting "things" to say "Every website uses HTML in order to display text or images on sites"